modules/web/js/ballerina/views/ballerina-file-editor.jsx

```diff
@@ -181,6 +187,29 @@ class BallerinaFileEditor extends React.Component {
     }
 
     /**
+     * Adds relevent imports needed to be automatically imported When a node (eg: a function invocation) is dragged in
+     * @param {Node} node the node added
+     */
+    addAutoImports(node) {
+        if (!TreeUtils.isAssignment(node) || !TreeUtils.isInvocation(node.getExpression())) {
```

Comment:
we need to check for var def statements also here

Reply:
Um. Does dragged in invocations create var def statements that require new imports to be added?  Aren't they always assignments?

Reply:
General tool palette has var def statements. But transform is assignments. Since we are making it generic, should add both.

Reply:
Got it. Updated PR